.hound.yml

```diff
@@ -276,3 +276,9 @@ Style/UnneededPercentQ:
 
 Metrics/AbcSize:
   Enabled: false
+
+ruby:
+  config_file: ruby-config.yml
+  enabled: true
+  ignore_file: /spec
```

Comment:
This actually wants you to create a root file - something like `.houndignore` - and put the files in there .

Also, `/spec` is not something you can ignore.